pylint/checkers/utils.py

```diff
@@ -244,8 +244,8 @@ def is_func_decorator(node):
         if isinstance(parent, astroid.Decorators):
             return True
         if (parent.is_statement or
-                isinstance(parent, astroid.Lambda) or
-                isinstance(parent, (scoped_nodes.ComprehensionScope,
+                isinstance(parent, (astroid.Lambda,
```

Comment:
The build `TOXENV=pylint` show me `consider-merging-isinstance` here.
Should I revert it and use a `pylint:disable=consider-merging-isinstance`?